indico/queries/metrics.py

```diff
@@ -0,0 +1,102 @@
+from indico.client.request import GraphQLRequest
+from indico.types.metrics import SequenceMetrics
+import json
+
```

Comment:
We can probably add a GetModelMetrics query here that first gets the model group and type the gets the appropriate model group metric by type. Maybe make a hash of metrics by task_type so it's easy to resolve in the continuation.

Reply:
Added the GetModelGroupMetrics query in the continuation style as prescribed